lib/head.js

```diff
@@ -81,7 +85,7 @@ function unique () {
           } else {
             const category = h.props[metatype]
             const categories = metaCategories[metatype] || new Set()
-            if (categories.has(category)) return false
+            if (categories.has(category) && !ALLOWED_DUPLICATES.includes(category)) return false
```

Comment:
Let's use `.indexOf` here, since I'm pretty sure people will run into issues with older browsers otherwise.

Reply:
yes, you' re right. Will fix this in a minute. Thanks for the feedback.